htmd/parameterization/dihedral.py

```diff
@@ -335,23 +287,33 @@ def _optimize_random_search(self, vector):
 
         return best_vector
 
+    def _updateDihedrals(self, parameters, keys, vector):
```

Comment:
For consistency, the name should be `_vectorToParams`. So it is obvious, it is doing opposite transformation to  `_paramsToVector`

Reply:
Okay I will rename it.